src/keycloak/chart/templates/uds-package.yaml

```diff
@@ -37,6 +49,14 @@ spec:
         remoteNamespace: "*"
         port: 8080
 
+      - description: "Keycloak ambient backchannel access"
+        direction: Ingress
+        selector:
+          app.kubernetes.io/name: keycloak
+        # Allow access from anything in cluster using an empty namespace selector
+        remoteNamespace: "*"
+        port: 15008
+
```

Comment:
The operator should automatically add 15008 to the policy right above this, see [here](https://github.com/defenseunicorns/uds-core/blob/main/src/pepr/operator/controllers/network/policies.ts#L154-L177).

Reply:
Fixed. Thanks for spotting this.